best-practice-authorisation.md

```diff
@@ -66,12 +66,11 @@ A short lived JSON Web Token that may be used by a client to access privileged r
 
 ### Bearer Token
 
-Bearer tokens are longer lived in access tokens. They are passed from the Authorization Server to the client after successful authentication of client credentials, and contain an access token. Bearer tokens can be used to retrieve further access tokens from the Authorization Server once the original access token expires.
+Bearer tokens are longer lived then access tokens. They are passed from the Authorization Server to the client after successful authentication of client credentials, and contain an access token. Bearer tokens can be used to retrieve further access tokens from the Authorization Server once the original access token expires.
```

Comment:
This is also dealt with (along with several other typos) in PR #22 at https://github.com/AMWA-TV/nmos-api-security/pull/22/commits/18b7c8d5f147d284c59a49fd3be265ee13cc5e39#diff-4b56e8871126a03a6dd47de63051dfa3R69.